src/host/services/IMUService.h

```diff
@@ -21,20 +21,29 @@
   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
   THE SOFTWARE.
 */
-
+#pragma once
 #include <Adafruit_BNO055.h>
 #include "common/os/Task.h"
 #include "common/signalk/SKHub.h"
+#include <EEPROM.h>
```

Comment:
You only need this in the `.cpp` file so you should put it only there. Having unused `#include` just slows down compile times.

Reply:
slowly, slowly I will learn this.... :-)

Reply:
repaired